src/main/java/sirius/pasta/noodle/macros/EscapeJSMacro.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.pasta.noodle.macros;
+
+import parsii.tokenizer.Position;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.pasta.noodle.Environment;
+import sirius.pasta.noodle.compiler.CompilationContext;
+import sirius.pasta.noodle.sandbox.PublicAPI;
+import sirius.web.resources.Resources;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * Escapes all line breakes and ' within the given string.
+ */
+@Register
+@PublicAPI
+public class EscapeJSMacro extends BasicMacro {
```

Comment:
```suggestion
public class EscapeJsMacro extends BasicMacro {
```

Reply:
will do that in the IDE to also rename the file..